cyclonedx/model/component.py

```diff
@@ -346,8 +368,8 @@ def type(self) -> PatchClassification:
         """
         return self._type
 
-    @type.setter
-    def type(self, type_: PatchClassification) -> None:
+    @type_.setter
+    def type_(self, type_: PatchClassification) -> None:
```

Comment:
:memo: breaking change.


:question: purpose of this rename?

Reply:
Avoid pythonic keyword `type`

Reply:
i would say this does not apply on a data model class.
let's overrule this, if it was a rule.